package.json

```diff
@@ -12,6 +12,7 @@
     "test": "SASS_PATH=./src/assets/styles jest --watch",
     "test:changed": "SASS_PATH=./src/assets/styles jest -o",
     "test:coverage": "SASS_PATH=./src/assets/styles jest --watchAll=false --coverage",
+    "test:staged": "SASS_PATH=./src/assets/styles jest --findRelatedTests",
```

Comment:
Do we actually need to set SASS_PATH here? Isn't it already defined in .env?

Reply:
Hmm does jest use that?

Reply:
Oh yeah, maybe that was the reason. Maybe there is no need to add support for env if this is only needed variable

Reply:
yeah but I would be nice to have it in a file. Harder to read these scripts because they are bloated with 1 env variables that is repeated